.circleci/config.yml

```diff
@@ -12,12 +12,43 @@ jobs:
         environment:
           POSTGRES_USER: root
           POSTGRES_DB: arlo-test
+    working_directory: ~/arlo
     steps:
       - checkout
+      - run: sudo chown -R circleci:circleci /usr/local/bin
+      - run: sudo chown -R circleci:circleci /usr/local/lib/python3.7/site-packages
+      - restore_cache:
+          keys:
+            - yarn-packages-v3-{{ .Branch }}-{{ checksum "yarn.lock" }}
+      - restore_cache:
+          keys:
+            - pip-packages-v4-{{ .Branch }}-{{ checksum "Pipfile.lock" }}
+            - pip-packages-v4-{{ .Branch }}-
+            - pip-packages-v4-
+      - restore_cache:
+          keys:
+            - npm-packages-v6-{{ .Branch }}-{{ checksum "package-lock.json" }}
```

Comment:
is it possible to name these steps so that we can tell which one is which? this is what I see now: 
<img width="262" alt="Screen Shot 2020-09-08 at 2 44 19 PM" src="https://user-images.githubusercontent.com/530106/92530742-cc8dc480-f1e1-11ea-9f51-0f10a13096a4.png">

Reply:
I'll have to check.